### PR TITLE
PMM-13222 Fix server-autobuild-el7 pipeline after centos 7 eol.

### PR DESCRIPTION
**Pull request by talhabinrizwan**
PMM-13222

Link to the Feature Build: SUBMODULES-3659

### CODE REVIEW

File: build/docker/server/Dockerfile [hunk 1]
```diff
@@ -13,7 +13,14 @@ EXPOSE 80 443
 
 WORKDIR /opt
 
-RUN yum -y install epel-release && yum -y install ansible
+# Disable fastestmirror plugin and update repo URLs
+RUN sed -i 's/enabled=1/enabled=0/g' /etc/yum/pluginconf.d/fastestmirror.conf && \
+    sed -i -e 's/^\(mirrorlist\)/#\1/g' /etc/yum.repos.d/CentOS-Base.repo && \
+    sed -i -e 's/^#baseurl.*/baseurl=http:\/\/vault.centos.org\/centos\/\$releasever\/os\/\$basearch\//g' /etc/yum.repos.d/CentOS-Base.repo
```
Comment (ademidoff) [inline build/docker/server/Dockerfile:19]:
```suggestion
    sed -i -e 's|^#baseurl.*|baseurl=http://vault.centos.org/centos/\$releasever/os/\$basearch/|g' /etc/yum.repos.d/CentOS-Base.repo
```

Comment (ademidoff) [reply build/docker/server/Dockerfile:19]: Let's use pipes instead of backslashed for better readability :)

**Review by ademidoff — APPROVED** [review]

File: build/ansible/roles/pmm2-images/tasks/main.yml [hunk 2]
```diff
@@ -55,7 +55,7 @@
     name: "*"
     state: latest
     exclude: "ansible*"
-    disablerepo: percona-release-x86_64
+    disablerepo: pmm2-server
```
Comment (ademidoff) [inline build/ansible/roles/pmm2-images/tasks/main.yml:58]: 👍

**Review by BupycHuk — APPROVED** [review]
Should we do the same during upgrade process?

**Comment by ademidoff** [comment]
> Should we do the same during upgrade process?

I guess no if the repo files have already been patched by the time the upgrade starts, and yes otherwise.

**Pull request merged**